packages/miniflare/src/index.ts

```diff
@@ -1445,6 +1479,19 @@ export class Miniflare {
 		// all of `requiredSockets` as keys.
 		this.#socketPorts = maybeSocketPorts;
 
+		if (this.#maybeInspectorProxy !== undefined) {
+			// Try to get inspector port for worker
```

Comment:
Should this be?

```suggestion
			// Try to get inspector port for the workers
```

In other words, this is the port for the internal inspector on the workerd instance, which will be wrapped by the inspector proxy?

Reply:
Yes, thanks for the fix 🙂 

> In other words, this is the port for the internal inspector on the workerd instance, which will be wrapped by the inspector proxy?

Yes exactly 🙂